src/components/PriceLine.tsx

```diff
@@ -50,14 +51,21 @@ const PriceLine = ({
     );
     const opacity = React.useMemo(() => (isOverlapping ? opacityOnOverlap : ''), [isOverlapping, opacityOnOverlap]);
 
+    const isBarrierZero = parseFloat(price) === 0.00;
```

Comment:
```suggestion
    const isBarrierZero = parseFloat(price) === 0;
```
@akmal-deriv  even if you define a number with trailing zeros, they're dropped :) 
<img width="229" alt="Screenshot 2023-09-08 at 8 54 34 AM" src="https://github.com/binary-com/SmartCharts/assets/103177211/cf87e61b-8b58-410f-b576-ba16a53f10e1">